Core/RetroAchievements.cpp

```diff
@@ -246,7 +246,7 @@ static void event_handler_callback(const rc_client_event_t *event, rc_client_t *
 		rc_client_user_game_summary_t summary;
 		rc_client_get_user_game_summary(g_rcClient, &summary);
 
-		std::string message = StringFromFormat(ac->T("%d achievements"), summary.num_unlocked_achievements);
+		std::string message = ApplySafeSubstitutions(ac->T("%d achievements, %d points"), summary.num_unlocked_achievements, summary.points_unlocked);
```

Comment:
Was this meant to stay `%d`?

-[Unknown]

Reply:
Uh, no, oops.